presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidIntegrationSmokeTest.java

```diff
@@ -273,4 +273,50 @@ public void testFilteringForTablesAndColumns()
         actualColumns = computeActual("DESCRIBE " + datasourceB);
         Assert.assertEquals(actualColumns, expectedColumns);
     }
+
+    @Test
+    public void testAggregationPushdown()
+    {
+        assertPushedDown("SELECT count(*) FROM orders");
+
+        // for varchar only count is pushed down
+        assertPushedDown("SELECT count(comment) FROM orders");
+
+        // for timestamp
+        assertPushedDown("SELECT count(__time) FROM orders", "select count(orderdate) FROM orders");
+        assertPushedDown("SELECT min(__time) FROM orders", "select min(cast(orderdate as timestamp(3))) FROM orders");
+        assertPushedDown("SELECT max(__time) FROM orders", "select max(cast(orderdate as timestamp(3))) FROM orders");
+
+        // for double
+        assertPushedDown("SELECT count(totalprice) FROM orders");
+        assertPushedDown("SELECT min(totalprice) FROM orders group by custkey");
+        assertPushedDown("SELECT max(totalprice) FROM orders");
+        assertPushedDown("SELECT sum(totalprice) FROM orders");
+        assertPushedDown("SELECT avg(totalprice) FROM orders");
+        assertPushedDown("SELECT stddev(totalprice) FROM orders");
+        assertPushedDown("SELECT stddev_samp(totalprice) FROM orders");
+        assertPushedDown("SELECT stddev_pop(totalprice) FROM orders");
+        assertPushedDown("SELECT variance(totalprice) FROM orders");
+        assertPushedDown("SELECT var_samp(totalprice) FROM orders");
+        assertPushedDown("SELECT var_pop(totalprice) FROM orders");
+
+        // for bigint
+        assertPushedDown("SELECT count(shippriority) FROM orders");
+        assertPushedDown("SELECT min(shippriority) FROM orders group by custkey");
+        assertPushedDown("SELECT max(shippriority) FROM orders");
+        assertPushedDown("SELECT sum(shippriority) FROM orders");
+        assertPushedDown("SELECT avg(shippriority) FROM orders");
+        assertPushedDown("SELECT stddev(shippriority) FROM orders");
+        assertPushedDown("SELECT stddev_samp(shippriority) FROM orders");
+        assertPushedDown("SELECT stddev_pop(shippriority) FROM orders");
+        assertPushedDown("SELECT variance(shippriority) FROM orders");
+        assertPushedDown("SELECT var_samp(shippriority) FROM orders");
+        assertPushedDown("SELECT var_pop(shippriority) FROM orders");
```

Comment:
Add tests for std/var on empty and 1-row tables.

Reply:
Added the tests but they are failing for now so commented it out until we figure out what do we want to about cases where the presto behavior is different from underlying connector's behavior.

Reply:
Uncommented the tests